### PR TITLE
Brings back Security Medic

### PR DESCRIPTION
**Pull request by faaaay**
## About The Pull Request

Security Medic was removed in the latest upstream merge. This undoes that, [as requested by the person who PRed said merge](https://github.com/Bubberstation/Bubberstation/pull/236). I've tested that the role can spawn, and that all sprites function.

## Changelog

:cl:
fix: un-deleted the Security Medic
/:cl:

### CODE REVIEW

**Comment by faaaay** [comment]
god damn you, ss13, i specifically didn't commit the changed config files.

**Comment by faaaay** [comment]
wait we don't even need to edit voidraptor thanks to area spawns

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm
Comment (ReturnToZender) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:1]: Why is this here? This literally just deletes alt titles for scientist.

Comment (faaaay) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm:1]: i have no idea how this happened

**Comment by ReturnToZender** [comment]
Make this modular, please!

**Comment by faaaay** [comment]
> Make this modular, please!

I thought I did. Edits are marked when necessary, and files are in modular_zubbers.

**Comment by nevimer** [comment]
I'll testmerge then merge once CI passes

**Comment by faaaay** [comment]
> I'll testmerge then merge once CI passes

no idea why CI is failing, it was fine initially and builds/runs locally

**Comment by nevimer** [comment]
It's failing become of some missing sprites / icon states.

**Comment by faaaay** [comment]
ah fuck, my bad

**Comment by nevimer** [comment]
needs to update to master now, its got merge conflicts

**Comment by faaaay** [comment]
bingo!

**Comment by nevimer** [comment]
`Error: The include at line 7154 is out of order (#include "modular_zubbers\code\modules\sec_haul\code\security_medic\secmed_clothes.dm", expected #include "modular_zubbers\code\modules\research\designs\medical_designs.dm")`

just rebuild locally and push the new tgstation.dme

**Comment by faaaay** [comment]
Voila!

**Comment by nevimer** [comment]
testmerge time

**Comment by faaaay** [comment]
wait, why is this _now_ giving CI errors?

File: modular_zubbers/code/datums/id_trim/jobs.dm [hunk 1]
```diff
@@ -35,4 +53,4 @@
 
 /datum/id_trim/job/nanotrasen_consultant
 	minimal_wildcard_access = list()
-	template_access = list(ACCESS_CAPTAIN) //NTRep can no longer change to any other ID trim
+	template_access = list(ACCESS_CAPTAIN) //NTRep can no longer change to any other ID trim
```
Comment (Cyprex) [inline modular_zubbers/code/datums/id_trim/jobs.dm:56]: unrelated change

**Comment by Cyprex** [comment]
CI needs to be figured out, I'll poke it and make it rerun tests

File: tgstation.dme [hunk 2]
```diff
@@ -7177,8 +7181,12 @@
 #include "modular_zubbers\code\modules\pai\ghost_pai.dm"
 #include "modular_zubbers\code\modules\paperwork\fax.dm"
 #include "modular_zubbers\code\modules\projectiles\guns\energy\pulse.dm"
+#include "modular_zubbers\code\modules\protected_roles\code\antag_restricted_jobs.dm"
 #include "modular_zubbers\code\modules\research\designs\comp_board_designs.dm"
 #include "modular_zubbers\code\modules\research\designs\medical_designs.dm"
+#include "modular_zubbers\code\modules\sec_haul\code\security_medic\secmed_clothes.dm"
+#include "modular_zubbers\code\modules\sec_haul\code\security_medic\security_medic.dm"
+#include "modular_zubbers\code\modules\research\designs\medical_designs.dm"
```
Comment (Cyprex) [inline tgstation.dme:7189]:
```suggestion
```
You accidently duplicated this line

File: modular_zubbers/code/datums/id_trim/jobs.dm (hunk 1 above)
Comment (Cyprex) [reply modular_zubbers/code/datums/id_trim/jobs.dm:56]: Actually this whole file is unrelated changes- this upstream merge seems broken: https://github.com/Bubberstation/Bubberstation/pull/268/commits/298feaa3b8cad315c347bfc326d820f29118cbb2

File: tgstation.dme (hunk 2 above)
Comment (faaaay) [reply tgstation.dme]: apologies, you're right

**Comment by faaaay** [comment]
> CI needs to be figured out, I'll poke it and make it rerun tests

Any idea what's messing with CI?
